base/channels.jl

```diff
@@ -105,6 +129,12 @@ function Channel(func::Function; ctype=Any, csize=0, taskref=nothing)
     isa(taskref, Ref{Task}) && (taskref[] = task)
     return chnl
 end
+function Channel{T}(f::Function, sz=0) where T
```

Comment:
This causes a method overwrite warning. This signature can be `Channel{T}(f::Function, sz)`.

Reply:
Oops, sorry! Fixed in https://github.com/JuliaLang/julia/pull/32818

Thanks, good catch! Sorry I didn't notice (the error comes up while building and it's easy to miss. I'll keep an eye out for that in the future!) :)